README.md

```diff
@@ -8,24 +8,34 @@ The following instructions are for running on a Windows machine. If you have acc
 # Installation
 Install the latest version of NodeJS at https://nodejs.org/
 
-Open Powershell, navigate to the directory where the repo was cloned, and use Node Package Manager to install Puppeteer.
-    
-    npm i puppeteer --save
+Open Powershell, navigate to the directory where the repo was cloned, and use Node Package Manager to install any dependencies (like Puppeteer).
 
-# Running the Script
+```sh
+npm i
+```
 
-This script uses a large amount of CPU and RAM. I was not able to run all 1000 browser sessions on one machine with 24GB of RAM. I recruited a friend to help, and he ran a couple instances of the script as well.
-
-Open the VstheWorld.js file in a text editor, and update the `numberOfSessions` value at the top of the file to the number of browser sessions your machine can handle. This might take some trial and error. You can also increase the `timeoutMilliseconds` value if the browsers time out before all 1000 sessions are reached.
+# Usage
 
+## Start the Game
 Open a game in streamer mode, and join the lobby as a player. To conserve PC resources, I joined the game on my phone. Note the room code, because we need it for the script. Start the game, then pause it to give the script time to run.
 
+## Running the Script
 I recommend opening Task Manager to keep an eye on your PC's resources when running the script.
 
 Open Powershell and navigate to the directory where the VstheWorld.js file is located. Execute the script using the following command, assuming the room code is ABCD:
 
-    node VstheWorld.js ABCD
+```sh
+npm start ABCD
+```
+
+This script uses a large amount of CPU and RAM. I was not able to run all 1000 browser sessions on one machine with 24GB of RAM. I recruited a friend to help, and he ran a couple instances of the script as well.
+
+You can also specify how many sessions your machine can handle. This might take some trial and error. You can also increase the timeout value (in milliseconds) if the browsers time out before all 1000 sessions are reached.
+
+```sh
+npm start ABCD -- --sessions 500 --timeout 2000000
```

Comment:
@eskfung is there an extra set of dashes on this line after the room code? Thanks for the contribution. I plan to test it later this evening.

Reply:
There is. If you are invoking the script directly with node, these are flagged arguments in the way you'd expect:

```sh
node VstheWorld.js ABCD --sessions 500 --timeout 20000
```

In my experience, since npm is helping you manage dependencies anyway, it's helpful to declare a [script](https://github.com/bprussell/VstheWorld/pull/4/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R5) that is a little easier to start. However, flagged arguments need an extra `--` to make sure they get passed through calling the script to the underlying process.

Reply:
So how does `npm start` know what script to run? Is it just because there's only one script in that path? I'm not a node developer by trade so I'm not familiar with this.

Reply:
Nevermind, I checked out your link and I understand now. Cool!